blockstore/splitstore/markset_badger.go

```diff
@@ -46,39 +48,12 @@ func NewBadgerMarkSetEnv(path string) (MarkSetEnv, error) {
 	return &BadgerMarkSetEnv{path: msPath}, nil
 }
 
-func (e *BadgerMarkSetEnv) Create(name string, sizeHint int64) (MarkSet, error) {
+func (e *BadgerMarkSetEnv) create(name string, sizeHint int64) (*BadgerMarkSet, error) {
 	path := filepath.Join(e.path, name)
 
-	// clean up first
-	err := os.RemoveAll(path)
```

Comment:
Not related to this patch, but this seems scary...

Reply:
Maybe fine, but I'd consider giving all "transient" datastores a common "tmp-" prefix.

Reply:
Probably better to just put it in the markSetEnv so that the entire directory is tmp labelled.

Reply:
Also, I extract the open/close to precisly have the `Transient` in the function name to make it clear that this really is temporary data.

Reply:
I added the `".tmp"` suffix to all markset names.